### DISCUSSION TITLE
`set -o vi`

### DISCUSSION THREAD

**Discussion by FraserLee** [xonsh/xonsh#5100] [Q&A] [answered]
Hi. Does xonsh support vi style command editing? I can't find anything about it in the documentation.

Cheers

**Comment by FraserLee** [comment]
Nevermind, found this https://xon.sh/api/_autosummary/cmd/xonsh.environ.html?highlight=vi#xonsh.environ.AsyncPromptSetting.VI_MODE

My bad!

**Comment by anki-code** [comment, marked as answer]
The better link is https://xon.sh/envvars.html#vi-mode
Welcome! :)
